Assets/BossRoom/Scripts/Client/UI/HeroActionBar.cs

```diff
@@ -197,6 +197,11 @@ void Update()
             // If we have another player selected, see if their aliveness state has changed,
             // and if so, update the interactiveness of the basic-action button
 
+            if (Input.GetKeyUp(KeyCode.Alpha4))
```

Comment:
any reason we're not managing this the same way we're managing other shortcuts? Why the special case here?

Reply:
You mean why is this not included inside `ClientInputSender`? The other actions (1-3) are sent as request RPCs from `ClientInputSender`, and on top of that we've also added `OnButtonDown` callbacks here for UI button clicks. We've simply just not added a response from input on Alpha4, and I certainly think it should be kept here instead of `ClientInputSender`, since it's just enabling/disabling a GameObject.